pkg/workflow/argo.go

```diff
@@ -33,7 +33,8 @@ const (
 )
 
 var (
-	timeout int64 = 3600
+	timeout            int64 = 3600
```

Comment:
We should convert these to `time.time` for readability

Reply:
Will do